xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -283,7 +282,7 @@ bool CGUIDialogVideoInfo::OnAction(const CAction &action)
   return CGUIDialog::OnAction(action);
 }
 
-void CGUIDialogVideoInfo::SetUserrating(int userrating)
+void CGUIDialogVideoInfo::SetUserrating(int userrating) const
```

Comment:
Are you sure about this? Line 291 modifies `m_movieItem` so the call to `CFileITem::GetVidoeInfoTag()` requires a non-cost class member.